common/beerocks/bcl/source/beerocks_utils.cpp

```diff
@@ -310,53 +310,3 @@ std::string utils::dump_buffer(uint8_t *buffer, size_t len)
     }
     return hexdump.str();
 }
-
-void utils::hex_dump(const std::string &description, uint8_t *addr, int len,
-                     const char *calling_file, int calling_line)
-{
-    int16_t i;
-    char ascii_chars[17]        = {};
-    uint8_t *pc                 = addr;
-    auto calling_file_str       = std::string(calling_file);
-    const auto caller_file_name = calling_file_str.substr(calling_file_str.rfind('/') + 1);
-    std::stringstream print_stream;
-
-    // Process every byte in the data.
-    for (i = 0; i < len; i++) {
-        // Multiple of 16 means new line (with line offset).
-
-        if ((i % 16) == 0) {
-            // Just don't print ASCII for the zeroth line.
-            if (i != 0) {
-                print_stream << "   " << std::string(ascii_chars) << std::endl;
```

Comment:
We're really loosing a lot of features here: the ASCII representation, the indentation, the offset... It might make sense to keep the implementation of hex_dump and just remove the caller part.